session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -173,7 +191,39 @@ public synchronized TSExecuteBatchStatementResp insertBatch(RowBatch rowBatch)
     }
   }
 
-  public synchronized TSStatus insert(String deviceId, long time, List<String> measurements,
+  /**
+   * insert data in batch format, which can reduce the overhead of network
+   */
+  public TSStatus insertInBatch(List<String> deviceIds, List<Long> times,
+      List<List<String>> measurementsList,
+      List<List<String>> valuesList)
+      throws IoTDBSessionException {
+    // check params size
+    int len = deviceIds.size();
+    if(len != times.size() || len != measurementsList.size() || len != valuesList.size()){
+      throw new IllegalArgumentException("deviceIds, times, measurementsList and valuesList 's size should be equal");
```

Comment:
```suggestion
      throw new IllegalArgumentException("deviceIds, times, measurementsList and valuesList's size should be equal");
```

Reply:
Fixed~

Reply:
Thanks for reviewing my code :D
I will update change list and code style